markdown/dev/reference/api/path/length/en.md

```diff
@@ -14,36 +14,29 @@ float path.length()
 
 <Example caption="Example of the Path.length() method">
 ```js
-({ Point, points, Path, paths, macro, part }) => {
+({ Point, points, Path, paths, macro, utils, part }) => {
 
   points.A = new Point(45, 60)
   points.B = new Point(10, 30)
   points.BCp2 = new Point(40, 20)
   points.C = new Point(90, 30)
   points.CCp1 = new Point(50, -30)
-  
-  paths.example = new Path()
+
+  paths.AB = new Path()
     .move(points.A)
     .line(points.B)
+
+  paths.BC = new Path()
+    .move(points.B)
     .curve(points.BCp2, points.CCp1, points.C)
-  
-  macro("pd", {
-    path: paths.example,
-    d: -20
-  })
-  
-  macro("pd", {
-    path: new Path().move(points.B).line(points.A),
-    d: 10
-  })
-  
-  macro("pd", {
-    path: new Path().move(points.B).curve(points.BCp2, points.CCp1, points.C),
-    d: -10
-  })
+
```

Comment:
I'm not sure why you felt the need to replace the macros, but we should refrain from hard-coding units in our examples.

The macros will automatically format the value in the units the user requested. So I suggest we keep them

Reply:
I removed the macros because I didn't want to confuse the user by presenting an "alternate" way to display path lengths (since the goal of the document was to show `Path.length()` in action). However, I'll re-add the macros and do the example a different way.